### ISSUE TITLE
Feature request: Filter by window title

### ISSUE THREAD

**Issue by EidolonForgery** [aardappel/procrastitracker#96]
A cursory examination of the issues for this repo state that the utility to pull the URL from firefox isn't working because, basically, Mozilla broke it. However, something else that could be useful - very useful, honestly - would be the ability to auto-tag based on some sort of filtering mechanism. For instance, automatically tagging "Youtube" as "time wasted". Is there a way to do this currently, and if not, could it be implemented?

**Comment by aardappel** [comment]
Yup, agree that be very useful to have, but do not have the bandwidth to work on that currently.
